desktop/core/src/desktop/lib/raz/raz_client_test.py

```diff
@@ -410,12 +405,28 @@ def test_handle_raz_ha(self):
           verify=False
         )
         assert_equal(raz_response.status_code, 200)
+        assert_equal(requests_post.call_count, 1)
+        requests_post.reset_mock()
 
-        # When no RAZ URL is healthy
-        requests_post.return_value = Mock(status_code=404)
+        # HA mode - When RAZ instance1 is unhealthy and RAZ instance2 is healthy
+        requests_post.side_effect = [Mock(status_code=404), Mock(status_code=200)]
+        raz_response = client._handle_raz_ha(self.raz_urls_ha, auth_handler=HTTPKerberosAuth(), data=request_data, headers={})
 
-        client = RazClient(self.raz_urls_ha, 'kerberos', username=self.username, service="s3", service_name="cm_s3", cluster_name="cl1")
+        requests_post.assert_called_with(
+          'https://raz_host_2.gethue.com:8080/api/authz/s3/access?doAs=gethue',
+          auth=HTTPKerberosAuth(),
+          headers={}, 
+          json=request_data, 
+          verify=False
+        )
+        assert_equal(raz_response.status_code, 200)
+        assert_equal(requests_post.call_count, 2)
+        requests_post.reset_mock()
+
+        # When no RAZ instance is healthy
+        requests_post.return_value = Mock(status_code=404)
         raz_response = client._handle_raz_ha(self.raz_urls_ha, auth_handler=HTTPKerberosAuth(), data=request_data, headers={})
 
         assert_equal(raz_response, None)
-
+        assert_equal(requests_post.call_count, 2)
+        requests_post.reset_mock()
```

Comment:
can we remove the last `requests_post.reset_mock()` ?

Reply:
Done